### ISSUE TITLE
折线图如何实现不同的区间值绘制不同

### ISSUE THREAD

**Issue by iscmd** [apache/echarts#19563]
### What problem does this feature solve?

这是需要实现的效果图
![WHGC72W{ZM3H3Z7~DANQMPF](https://github.com/apache/echarts/assets/48424960/93b9faa4-e42e-46be-a93a-69b7c04b8d22)

### What does the proposed API look like?

这是需要实现的效果图
![WHGC72W{ZM3H3Z7~DANQMPF](https://github.com/apache/echarts/assets/48424960/93b9faa4-e42e-46be-a93a-69b7c04b8d22)

**Comment by MatthiasMert** [comment]
[Demo](https://echarts.apache.org/examples/en/editor.html?c=area-time-axis&code=DYUwLgBARghgziCBeCBqAdiA7hAIjMEACgEYBOADgoBoIzaBmASgG4AoUSAe03wE9kEAEwAWCACoIDAGwAGWRIgl5s9pwgATAjEEBtXbAS0AsgQAWAOgBOMdBq4BbIk0UN5AXXfsAZlysQidQBLQRIWCBCAHikAVlUI1FQXAG82CAh1dC4cFEwcfEIiIkNEVBQeEH4mVjTNbQsABwBXODMiXQxsk3NrLia7ItMwSxs7R2cIAFoIWQsYl0khBVQ6sBhdEOmSd11tpncagF82LgawIJ5BVPSADwBBG6C4AC4Ia_SIMD4GkFeAcnODhAf2otXScAawCCYAAMkFMK93h8IK1sq8wFYmiBQcjjulDjiIHwHk9EWDPt9fhA_gA3GDALEg8kQqGw-FU5KorCvbz0hB4iAE2oIKxBEAvCC6clIj5fH7_KGYJnI8EOLhcYbozHY8ngvgOKBcYD_LJKwnImBWEAwADKX1AiKFKtWMFeWjWFjgUIAxsRZNQSAB2aRMclOj4y9Jyql_RXA80fOBqjVmLVYhN6g1Gk0VZUqy3Wu18B1vcPI92ul2en3EIMiagMOKh3G1dxsQ4sIA)

The [xAxis interval](https://echarts.apache.org/en/option.html#xAxis.splitLine.interval) property works only for category axis.

**Comment by iscmd** [comment]
> [Demo](https://echarts.apache.org/examples/en/editor.html?c=area-time-axis&code=DYUwLgBARghgziCBeCBqAdiA7hAIjMEACgEYBOADgoBoIzaBmASgG4AoUSAe03wE9kEAEwAWCACoIDAGwAGWRIgl5s9pwgATAjEEBtXbAS0AsgQAWAOgBOMdBq4BbIk0UN5AXXfsAZlysQidQBLQRIWCBCAHikAVlUI1FQXAG82CAh1dC4cFEwcfEIiIkNEVBQeEH4mVjTNbQsABwBXODMiXQxsk3NrLia7ItMwSxs7R2cIAFoIWQsYl0khBVQ6sBhdEOmSd11tpncagF82LgawIJ5BVPSADwBBG6C4AC4Ia_SIMD4GkFeAcnODhAf2otXScAawCCYAAMkFMK93h8IK1sq8wFYmiBQcjjulDjiIHwHk9EWDPt9fhA_gA3GDALEg8kQqGw-FU5KorCvbz0hB4iAE2oIKxBEAvCC6clIj5fH7_KGYJnI8EOLhcYbozHY8ngvgOKBcYD_LJKwnImBWEAwADKX1AiKFKtWMFeWjWFjgUIAxsRZNQSAB2aRMclOj4y9Jyql_RXA80fOBqjVmLVYhN6g1Gk0VZUqy3Wu18B1vcPI92ul2en3EIMiagMOKh3G1dxsQ4sIA)
> 
> The [xAxis interval](https://echarts.apache.org/en/option.html#xAxis.splitLine.interval) property works only for category axis.

请问参考是xAxis的splitLine的interval属性吗

**Comment by iscmd** [comment]
> [Demo](https://echarts.apache.org/examples/en/editor.html?c=area-time-axis&code=DYUwLgBARghgziCBeCBqAdiA7hAIjMEACgEYBOADgoBoIzaBmASgG4AoUSAe03wE9kEAEwAWCACoIDAGwAGWRIgl5s9pwgATAjEEBtXbAS0AsgQAWAOgBOMdBq4BbIk0UN5AXXfsAZlysQidQBLQRIWCBCAHikAVlUI1FQXAG82CAh1dC4cFEwcfEIiIkNEVBQeEH4mVjTNbQsABwBXODMiXQxsk3NrLia7ItMwSxs7R2cIAFoIWQsYl0khBVQ6sBhdEOmSd11tpncagF82LgawIJ5BVPSADwBBG6C4AC4Ia_SIMD4GkFeAcnODhAf2otXScAawCCYAAMkFMK93h8IK1sq8wFYmiBQcjjulDjiIHwHk9EWDPt9fhA_gA3GDALEg8kQqGw-FU5KorCvbz0hB4iAE2oIKxBEAvCC6clIj5fH7_KGYJnI8EOLhcYbozHY8ngvgOKBcYD_LJKwnImBWEAwADKX1AiKFKtWMFeWjWFjgUIAxsRZNQSAB2aRMclOj4y9Jyql_RXA80fOBqjVmLVYhN6g1Gk0VZUqy3Wu18B1vcPI92ul2en3EIMiagMOKh3G1dxsQ4sIA)
> 
> The [xAxis interval](https://echarts.apache.org/en/option.html#xAxis.splitLine.interval) property works only for category axis.

我看demo中是将series的data分割成两部分

**Comment by MatthiasMert** [comment]
Yes I tried to get as close as possible to your image. Please spacify more in detail what you want to achieve if not solved.
